model/m3api/m3path.pb.go

```diff
@@ -226,9 +227,9 @@ func (m *PathContextResponseMsg) GetMaxDist() int32 {
 }
 
 type PathNodesRequestMsg struct {
-	PathCtxId            int32    `protobuf:"varint,1,opt,name=path_ctx_id,json=pathCtxId,proto3" json:"path_ctx_id"`
-	Dist                 int32    `protobuf:"varint,2,opt,name=dist,proto3" json:"dist"`
-	ToDist               int32    `protobuf:"varint,3,opt,name=to_dist,json=toDist,proto3" json:"to_dist"`
+	PathCtxId            int32    `protobuf:"varint,1,opt,name=path_ctx_id,json=pathCtxId,proto3" json:"path_ctx_id" query:"path_ctx_id"`
+	Dist                 int32    `protobuf:"varint,2,opt,name=dist,proto3" json:"dist" query:"dist"`
+	ToDist               int32    `protobuf:"varint,3,opt,name=to_dist,json=toDist,proto3" json:"to_dist" query:"to_dist"`
```

Comment:
to parse the query string, we may need to modify the generated `pb.go`. Not sure we wanna do that, do the parse field by field manually can be another option